VimeoNetworking/Sources/NSError+Extensions.swift

```diff
@@ -192,20 +191,20 @@ public extension NSError
         /// Returns the api error code of the failing response, if available
     public var vimeoServerErrorCode: Int?
     {
-        if let errorCode = (self.userInfo[self.dynamicType.VimeoErrorCodeKeyLegacy] as? NSNumber)?.integerValue
+        if let errorCode = (self.userInfo[type(of: self).VimeoErrorCodeKeyLegacy] as? NSNumber)?.intValue
```

Comment:
Since `userInfo` is now `[AnyHashable: Any]`, I think we can eliminate the whole NSNumber intValue thing